package/rpm/post_install_section

```diff
@@ -1,9 +1,6 @@
-# Add the "decisionengine" user and group if they do not exist
-getent group decisionengine >/dev/null ||
-    groupadd -r  decisionengine
-getent passwd  decisionengine >/dev/null || \
-    useradd -r -g  decisionengine -d /var/lib/decisionengine \
-    -c "Decision Engine user" -s /sbin/nologin -m decisionengine
+# make sure our home area makes sense since we have a dynamic id
+chown decisionengine:decisionengine %{_sharedstatedir}/decisionengine
+chmod 750 /var/lib/decisionengine
```

Comment:
Why not use %{_sharedstatedir} also here?

Reply:
Fixed